### PR TITLE
Fix navigation and fetch less data from GitHub in dev mode

### PR DESCRIPTION
**Pull request by rsvoboda**
- Fix navigation to status sections
- Fetch less data from GitHub in dev mode

### CODE REVIEW

File: src/main/java/io/quarkus/status/IssuesService.java [hunk 1]
```diff
@@ -21,12 +22,14 @@
     public static final String BUG_LABEL = "kind/bug";
     private static final String ENHANCEMENT_NAME = "Enhancements";
     public static final String ENHANCEMENT_LABEL = "kind/enhancement";
-    private static final LocalDate ISSUES_STATS_START = LocalDate.of(2019, 1, 1);
     private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM");
 
     @Inject
     GitHubService gitHubService;
 
+    @ConfigProperty(name = "quarkus-status.issues.stats.start", defaultValue = "2019-01-01")
```
Comment (gsmet) [inline src/main/java/io/quarkus/status/IssuesService.java:30]:
```suggestion
    @ConfigProperty(name = "status.issues.stats.start", defaultValue = "2019-01-01")
```

That's how the other is called (the token).

File: src/main/java/io/quarkus/status/IssuesService.java [hunk 2]
```diff
@@ -21,12 +22,14 @@
...
+    private LocalDate ISSUES_STATS_START;
```
Comment (gsmet) [inline src/main/java/io/quarkus/status/IssuesService.java:31]: It shouldn't be uppercase, it's not a constant.

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -9,3 +9,4 @@ quarkus.openshift.host=status.quarkus.io
 quarkus.openshift.annotations."kubernetes.io/tls-acme"=true
 quarkus.openshift.env.secrets=quarkus-status-token
 
+%dev.quarkus-status.issues.stats.start=2020-09-01
```
Comment (gsmet) [inline src/main/resources/application.properties:12]:
```suggestion
%dev.status.issues.stats.start=2020-09-01
```

**Review by gsmet — CHANGES_REQUESTED** [review]

**Comment by rsvoboda** [comment]
@gsmet PR is updated

**Review by gsmet — APPROVED** [review]

**Pull request merged**